notification-service/src/main/java/com/redhat/parodos/notification/validation/ValidNotificationMessageUserOrGroupValidator.java

```diff
@@ -0,0 +1,28 @@
+package com.redhat.parodos.notification.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import com.redhat.parodos.notification.dto.NotificationMessageCreateRequestDTO;
+
+import org.springframework.util.CollectionUtils;
+
+public class ValidNotificationMessageUserOrGroupValidator
+		implements ConstraintValidator<ValidNotificationMessageUserOrGroup, NotificationMessageCreateRequestDTO> {
+
+	@Override
+	public void initialize(ValidNotificationMessageUserOrGroup constraintAnnotation) {
+	}
+
+	@Override
+	public boolean isValid(NotificationMessageCreateRequestDTO notificationMessage,
+			ConstraintValidatorContext context) {
+		if (notificationMessage == null) {
+			return true;
```

Comment:
In this case we do not check whether usernames nor groupnames are empty

Reply:
Changed to return false